lib/IHP/Makefile.dist

```diff
@@ -106,6 +106,9 @@ build/bin:
 run-production: build/bin/RunProdServer build/Generated/Types.hs ## Run Production Server
 	build/bin/RunProdServer
 
+postgres: ## Starts the postgresql server
```

Comment:
Would be a bit better to have a message if the database is already running.
```
postgres: ## Starts the postgresql server
	@if [ -s build/db/state/postmaster.pid ]; then
		echo "Postgres is already running."
	else
		echo "Starting database"
		postgres -D build/db/state -k $$PWD/build/db -c "listen_addresses="
	fi
```

Reply:
Ah thats even better, added it!